### PR TITLE
Added code to display staff member content

### LINKED ISSUE #18: Implement Shortcode
- [ ] Process attributes to setup wp_query
- [ ] Process staff loop template and replace template tags with post content

### PR DESCRIPTION
**Pull request by brettshumaker**
Moved all shortcode content to new structure. @josiahgoff check to make
sure I registered the shortcode in the right place within the new
structure.

Closes #18

### CODE REVIEW

File: trunk/public/class-simple-staff-list-public.php [hunk 1]
```diff
@@ -46,11 +46,21 @@ class Simple_Staff_List_Public {
...
 	 * @param      string    $plugin_name       The name of the plugin.
 	 * @param      string    $version    The version of this plugin.
+	 * @param	   array	 $simple_staff_list_shortcode_atts		The default attributes for the [simple_staff_list] shortcode
 	 */
 	public function __construct( $plugin_name, $version ) {
 
 		$this->plugin_name = $plugin_name;
 		$this->version     = $version;
+		
+		$this->simple_staff_list_shortcode_atts = array(
```
Comment (josiahgoff) [inline trunk/public/class-simple-staff-list-public.php]: Should we declare this property above construct with the others?

**Comment by josiahgoff** [comment]
Yeah, looks like everything is in the right place.

File: trunk/public/class-simple-staff-list-public.php (hunk 1 above)
Comment (brettshumaker) [inline trunk/public/class-simple-staff-list-public.php]: Yeah, probably should. Thanks

File: trunk/public/class-simple-staff-list-public.php [hunk 2]
```diff
@@ -39,18 +39,34 @@ class Simple_Staff_List_Public {
...
+	 * @since 	1.2
+	 * @access 	private
+	 * @var		
+	 */
+	private $simple_staff_list_shortcode_atts = array(
+		  'single' => 'no',
+		  'group' => '',
+		  'wrap_class' => '',
+		  'order' => 'ASC',
+		);
```
Comment (brettshumaker) [inline trunk/public/class-simple-staff-list-public.php]: Like this @josiahgoff?

I also removed it from the constructor below.

**Pull request merged**